PHP 8.0 build is broken

composer.json states 8.0 is supported

```
    "php": "^8.0"
```

But buid is broken

```
/builddir/build/BUILD/aspect-0.1.0/aspect.c:31:1: warning: 'visibility' attribute ignored [-Wattributes]
 ZEND_API static void aspect_execute_ex(zend_execute_data *execute_data);
 ^~~~~~~~
/builddir/build/BUILD/aspect-0.1.0/aspect.c: In function 'aspect_execute_ex':
/builddir/build/BUILD/aspect-0.1.0/aspect.c:114:93: error: 'IS_NEVER' undeclared (first use in this function); did you mean 'IS_VAR'?
                     if (type_mask & ((uint32_t)1 << IS_VOID) || type_mask & ((uint32_t)1 << IS_NEVER)) {
                                                                                             ^~~~~~~~
                                                                                             IS_VAR
/builddir/build/BUILD/aspect-0.1.0/aspect.c:114:93: note: each undeclared identifier is reported only once for each function it appears in
make: *** [Makefile:208: aspect.lo] Error 1

```

P.S. 8.1 to 8.4 are OK (build + test suite)

--- a/aspect.c
+++ b/aspect.c
@@ -28,7 +28,7 @@ ZEND_DECLARE_MODULE_GLOBALS(aspect)
 
 static void (*original_zend_execute_ex)(zend_execute_data *execute_data);
 
-ZEND_API static void aspect_execute_ex(zend_execute_data *execute_data);
+static void aspect_execute_ex(zend_execute_data *execute_data);
 
 static void handle_memoize_functions(zend_execute_data *execute_data);
 
@@ -111,7 +111,11 @@ static void aspect_execute_ex(zend_execute_data *execute_data) {
                     zend_type return_type = ret_info->type;
                     uint32_t type_mask = return_type.type_mask;
 
-                    if (type_mask & ((uint32_t)1 << IS_VOID) || type_mask & ((uint32_t)1 << IS_NEVER)) {
+                    if (type_mask & ((uint32_t)1 << IS_VOID)
+#if PHP_VERSION_ID >= 80100
+                        || type_mask & ((uint32_t)1 << IS_NEVER)
+#endif
+                    ) {
                         // cannot memoize functions with void or never return types
                         // @TODO: Should this be an error/exception?
                         original_zend_execute_ex(execute_data);

